x/auth/mapper.go

```diff
@@ -71,7 +113,13 @@ func (am AccountKeeper) GetAccount(ctx sdk.Context, addr sdk.AccAddress) Account
 	if bz == nil {
 		return nil
 	}
+
+	strBz := string(bz)
```

Comment:
No need for a new variable here, just do it inline when we call `accountCache.getAccount`.

Reply:
No, you don't want to have two string conversions. This is how it should be imo, string conversion is not free.

Reply:
You're right, I didn't see it in `addAccount`.